src/libs/actions/Subscription.ts

```diff
@@ -279,6 +292,44 @@ function clearOutstandingBalance() {
     API.write(WRITE_COMMANDS.CLEAR_OUTSTANDING_BALANCE, null, onyxData);
 }
 
+function cancelBillingSubscription(cancellationReason: FeedbackSurveyOptionID, cancellationNote: string) {
+    const successData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: ONYXKEYS.NVP_PRIVATE_CANCELLATION_DETAILS,
+            value: [
+                ...cancellationDetails,
+                {
+                    errors: undefined,
+                },
```

Comment:
I think this is causing issues with the onyx db. It looks like this might be adding a "errors: undefined" json object every time the api is called, regardless of whether the api is successful or not. Doesn't seem right...

Reply:
Right, I removed it